python_apps/airtime_analyzer/setup.py

```diff
@@ -37,7 +39,7 @@
           'mock',
           'python-daemon',
           'requests>=2.7.0',
-          'rgain3',
+          'rgain3'
```

Comment:
The trailing `,` is helpful for anyone un-commenting the other dependencies

Reply:
Ah, okay. I thought it would cause issues.